File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <iostream>
#include <unordered_set>

class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_set<int> myHashSet;
        for (size_t i = 0; i < nums.size(); ++i) {
            if (myHashSet.contains(nums.at(i)))
            {
                return true;
            }
            myHashSet.insert(nums.at(i));
        }
        return false;
    }
};

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int val =0;
        for (int x = 0; x < nums.length+1; x++) {
            val ^=x;
        }
        for(int x: nums)
        {
            val ^=x;
        }
        return val;
    }
}
